
From: Hans Reiser <reiser@namesys.com>

This makes warning about unability to mount reiser4 more informative.

Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/fs/reiser4/init_super.c |    2 +-
 1 files changed, 1 insertion(+), 1 deletion(-)

diff -puN fs/reiser4/init_super.c~reiser4-fill_super-improve-warning fs/reiser4/init_super.c
--- 25/fs/reiser4/init_super.c~reiser4-fill_super-improve-warning	2004-09-20 11:06:18.872724056 -0700
+++ 25-akpm/fs/reiser4/init_super.c	2004-09-20 11:06:18.876723448 -0700
@@ -195,7 +195,7 @@ _INIT_(read_super)
 		brelse(super_bh);
 	} else {
 		if (!silent) {
-			warning("nikita-2608", "Wrong master super block magic.");
+			warning("nikita-2608", "%s: wrong master super block magic.", s->s_id);
 		}
 
 		/* no standard reiser4 super block found */
_
